packages/atomic/src/components/atomic-load-more-results/atomic-load-more-results.tsx

```diff
@@ -108,7 +109,10 @@ export class AtomicLoadMoreResults {
   }
 
   public render() {
-    if (!this.querySummaryState.hasResults) {
+    if (
+      !isAppLoaded(this.bindings.store) ||
```

Comment:
Is there * any * component for which the check on `isAppLoaded` should not be performed when rendering ?

Granted, for many of them it will always be true (eg: results list components). 

Should it be embedded in the `InitializeBindings` function ? Same way we check for generic component errors.

Reply:
From UX perspective, only the pagination should wait here, isAppLoaded will be true when the search is finished and the template components have rendered.

The search box, facet, header components (query summary. sort, etc) all load before the results, and some have their own placeholders. If we wait and display nothing, the page will look much more sluggish as nothing will be displayed for a while